test/e2e/pages/move-timeline.js

```diff
@@ -0,0 +1,17 @@
+import { Selector } from 'testcafe'
+
+import MoveDetailPage from './move-detail'
+
+class MoveTimelinePage extends MoveDetailPage {
+  constructor() {
+    super()
+
+    this.nodes = {
+      ...this.nodes,
+      timeline: Selector('.app-timeline'),
+      timelineItem: Selector('.app-timeline__item'),
```

Comment:
`timelineItems`? I'm guessing there will be many?

Reply:
mmkay